src/app/(public-pages)/disclaimer/page.tsx

```diff
@@ -1,18 +1,21 @@
+import Container from '@/components/Container'
 import PageHeader from '@/components/PageHeader'
 
 export const revalidate = 120
 
 export default async function Disclaimer() {
   return (
-    <>
-      <PageHeader heading="Disclaimer" />
-      <div className="pb-10">
-        <p className=" text-lrg text-gray-300">
-          Some deals may include affiliate links. Proceeds from these affiliate
-          links are invested back into the developer community through giveaways
-          on courses and other resources.
-        </p>
-      </div>
-    </>
+    <Container>
+      <main>
+        <PageHeader heading="Disclaimer" />
+        <div className="pb-10">
+          <p className=" text-lrg text-gray-300">
```

Comment:
I believe you meant `text-lg` rather than `text-lrg`.

Reply:
Another good catch! Fixed in latest commit!